cmd/preview/endpoints.go

```diff
@@ -46,11 +48,19 @@ func Endpoints(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			jsonContextBuffer := bytes.NewBuffer([]byte{})
+			if output == "json" {
```

Comment:
I know it's not part of the scope of the bug reported but I was wondering:

We support two output formats: `json` and `md` (besides the default one which would be something like: `plaintext` or `text`)

Would it make sense to slightly adjust this change to also fix `md`? I imagine the `md` output is useful to be injected in a Markdown document, but at the moment it would also output and inject the logs, making the feature kind of broken.

My proposal would be change this condition to be `if output != "text" {..}` and add `text` as valid output format.

I actually think we should add `plaintext` or `text` to all output flags, I've created an [issue about it](https://github.com/okteto/okteto/issues/3805) some time ago

Reply:
I don't want to block the PR, the change works and LGTM. If we can squeeze the change good, or I can create a follow-up issue (I didn't because I think the change could be very tiny, let me know!)

Reply:
IMO I think it is safe and has the sense to address your suggestion in this PR 👍

Reply:
It'd be great to extend them but I'll address in another PR. In fact there is a [current issue](https://github.com/okteto/okteto/issues/3859) for the md

Reply:
sounds good to me, already approved, so feel free to ship it 🚢